src/bitcoin-cli.cpp

```diff
@@ -887,7 +887,8 @@ static void ParseError(const UniValue& error, std::string& strPrint, int& nRet)
             strPrint += ("error message:\n" + err_msg.get_str());
         }
         if (err_code.isNum() && err_code.getInt<int>() == RPC_WALLET_NOT_SPECIFIED) {
-            strPrint += "\nTry adding \"-rpcwallet=<filename>\" option to bitcoin-cli command line.";
+            strPrint += " For the CLI, specify the \"-rpcwallet=<filename>\" option before the command";
+            strPrint += " (run \"bitcoin-cli -h\" for help or \"bitcoin-cli listwallets\" to see which wallets are currently loaded).";
```

Comment:
Where does `bitcoin-cli -h` show how to solve this problem specifically? Telling users to run that here might not be very helpful. The second part could be useful information though.

Reply:
@andrewtoth I've updated the error message, check if that looks better for you.